src/arch/bsdsocketinterf.cpp

```diff
@@ -175,7 +183,33 @@ CBSDSocketInterface::TSocketDescriptor CBSDSocketInterface::openUDPSendPort(char
           DEVLOG_WARNING("CBSDSocketInterface: setsockopt(IP_MULTICAST_IF) failed: %s\n", strerror(errno));
       }
     }
-#endif
+#else
+    if (paMCInterface) {
```

Comment:
I would prefer to have that somewhere else. If that is not possible at all please move it at least into a function so that the code stays readable.

Reply:
Done.